File: QuickStart/TreeView/Localization/GermanTreeViewLocalizationProvider.vb
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports Telerik.WinControls.UI

Namespace Telerik.Examples.WinControls.TreeView.Localization
	Public Class GermanTreeViewLocalizationProvider
		Inherits TreeViewLocalizationProvider
		Public Overrides Function GetLocalizedString(ByVal id As String) As String
			Select Case id
				Case TreeViewStringId.ContextMenuCollapse
					Return "Zusammenklappen"
				Case TreeViewStringId.ContextMenuDelete
					Return "Löschen"
				Case TreeViewStringId.ContextMenuEdit
					Return "Edieren"
				Case TreeViewStringId.ContextMenuExpand
					Return "Expandieren"
				Case TreeViewStringId.ContextMenuNew
					Return "Neu"
			End Select

			Return ""
		End Function
	End Class
End Namespace

File: QuickStart/TreeView/Localization/GermanTreeViewLocalizationProvider.cs
﻿using System;
using System.Collections.Generic;
using System.Text;
using Telerik.WinControls.UI;

namespace Telerik.Examples.WinControls.TreeView.Localization
{
    public class GermanTreeViewLocalizationProvider: TreeViewLocalizationProvider
    {
        public override string GetLocalizedString(string id)
        {
            switch (id)
            {
                case TreeViewStringId.ContextMenuCollapse:
                    return "Zusammenklappen";
                case TreeViewStringId.ContextMenuDelete:
                    return "Löschen";
                case TreeViewStringId.ContextMenuEdit:
                    return "Edieren";
                case TreeViewStringId.ContextMenuExpand:
                    return "Expandieren";
                case TreeViewStringId.ContextMenuNew:
                    return "Neu";
            }

            return "";
        }
    }
}
